pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -497,14 +490,16 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
         extract_dict['RealEstate_RestrictionOnLandownership'] = restrictions
         # End one restriction entry per theme
 
-        for item in extract_dict.get('Disclaimer', []):
+        for i, item in enumerate(extract_dict.get('Disclaimer', [])):
```

Comment:
Could there ever be more than two items here? 
Shoukd this be cheked an warned if there are more than two? (I know, it was not done before)

Reply:
OK lets leave it.